packages/components/src/search/autocompleters/countries.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import { computeSuggestionMatch } from './utils';
+
+/**
+ * A country completer.
+ * See https://github.com/WordPress/gutenberg/tree/master/packages/components/src/autocomplete#the-completer-interface
+ *
+ * @type {Completer}
+ */
+export default {
+	name: 'countries',
+	className: 'woocommerce-search__country-result',
+	options() {
+		return wcSettings.dataEndpoints.countries || [];
+	},
+	getOptionKeywords( country ) {
+		return [ decodeEntities( country.name ) ];
+	},
+	getOptionLabel( country, query ) {
+		const name = decodeEntities( country.name );
+		const match = computeSuggestionMatch( name, query ) || {};
+		return [
+			<span key="name" className="woocommerce-search__result-name" aria-label={ name }>
+				{ match.suggestionBeforeMatch }
+				<strong className="components-form-token-field__suggestion-match">
+					{ match.suggestionMatch }
+				</strong>
+				{ match.suggestionAfterMatch }
+			</span>,
```

Comment:
I don't think its in the design, but it might be cool to include the `<Flag />` component in a suggestion similar to how we show products autocompleter suggestions:

![screen shot 2018-12-13 at 11 10 15 am](https://user-images.githubusercontent.com/1922453/49902089-daf16400-fec7-11e8-8ffc-5e668c1322a4.png)

Reply:
Great suggestion! 

<img width="896" alt="screen shot 2018-12-13 at 11 35 36 am" src="https://user-images.githubusercontent.com/689165/49953032-3d982f00-fecb-11e8-936d-b8fe2ed1e951.png">

Reply:
Nice addition, and great to see our old friend the Flag component get some love!